### PR TITLE
Introduce rollup bundling

### PR DESCRIPTION
**Pull request by LennartWeihs**

### CODE REVIEW

File: libIndex.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export * from "./src/lib/form/util/validate.js"
+export { default as HiddenRadioBox } from "./src/lib/form/components/HiddenRadioBox.jsx"
```
Comment (franziskacomploi) [inline libIndex.js:2]: @LennartWeihs Just for clarficiation, would this need to be done manually?

File: rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      mainFields: ["module", "main", "jsnext:main", "browser"],
+      extensions,
+    }),
+    commonjs(),
+    postcss(),
+    babel({ extensions, runtimeHelpers: true }),
+    terser(),
+    // copy useful example files (e.g., css)
+    copy({
+      targets: [{ src: "src/styles/*", dest: "dist/styles" }],
```
Comment (franziskacomploi) [inline rollup.config.js:40]: @LennartWeihs It copies useful files so in case the user of the package wants to use these?

File: libIndex.js (hunk 1 above)
Comment (LennartWeihs) [reply libIndex.js:2]: @franziskapendzialek Yes, we'd have to update these manually. I did add a small shell script to do it for now, though

File: rollup.config.js (hunk 2 above)
Comment (LennartWeihs) [reply rollup.config.js:40]: @franziskapendzialek That would be my idea, so if the user wants to recreate our example, the necessary css is also part of the package

**Pull request merged**